packages/core/helper-plugin/lib/src/components/ReactSelect/utils/getSelectStyles.js

```diff
@@ -15,6 +15,8 @@ const getSelectStyles = (theme, error) => {
         border = `1px solid ${theme.colors.primary600} !important`;
       } else if (error) {
         border = `1px solid ${theme.colors.danger600} !important`;
+      } else if (state.isDisabled) {
+        border = `1px solid ${theme.colors.neutral200} !important`;
```

Comment:
Mh, these conditionals are getting a bit out of hand. What do you think about:

```js
let color = theme.colors.neutral200;

if (state.isFocused) {
  color = theme.colors.primary600;
} else if (error) {
  color = theme.colors.danger600;
}

border = `1px solid ${color} !important`;
```

This would allow us to drop two conditionals ...

Reply:
you're right I got mixed up!
I'm not sure we need to create 2 variables though (color & border) since I could remove some noise there
updated [here](https://github.com/strapi/strapi/pull/13687/commits/f383fbb6a7ed21c5014638c81ff516830ba5222e), wdyt?